src/txn/txn_recover.c

```diff
@@ -989,6 +994,18 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[])
         __wt_err(session, ret, "Recovery failed");
     }
 
+    /*
```

Comment:
The main intention of closing all the handles are due to the transaction information that didn't get reset until the btree write gen number is less than the base write generation. Add more details in the comment with the above details.

Reply:
Updated comments